### PR TITLE
chore(license): add Ratatui developers to license

### PR DESCRIPTION
**Pull request by joshka**
Updated the @fdehau's copyright to 2016-2022 and added "The Ratatui developers 2023"

This is in compliance with the license conditions (specifically the right to sublicense and the need to include the copyright notice):

Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the "Software"), to deal
in the Software without restriction, including without limitation the rights
to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
copies of the Software, and to permit persons to whom the Software is
furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.

### CODE REVIEW

**Review by orhun — APPROVED** [review]

**Comment by orhun** [comment]
Let's wait for approval from other maintainers for this one.

**Review by sayanarijit — APPROVED** [review]

**Review by mindoodoo — APPROVED** [review]

**Review by sophacles — APPROVED** [review]

**Pull request merged**
